src/test/ui/exhaustive_integer_patterns.stderr

```diff
@@ -10,11 +10,11 @@ note: lint level defined here
 LL | #![deny(unreachable_patterns)]
    |         ^^^^^^^^^^^^^^^^^^^^
 
-error[E0004]: non-exhaustive patterns: `128u8..=255u8` not covered
+error[E0004]: non-exhaustive patterns: `128u8..=::std::u8::MAX` not covered
```

Comment:
I agree that it's quite noisy, but considering that the compiler doesn't currently suggest importing `std::u8` if `u8::MAX` is not found, it could be confusing that it's not possible to directly copy and paste. Maybe we could start with this more verbose version and then cut it down after implementing a suggestion for importing `std::u8`;